input/pages/measure-conformance.md

```diff
@@ -823,6 +823,12 @@ In addition to the measure scoring, measures generally fall into two categories,
 
 1. The [`cqm-populationBasis`](StructureDefinition-cqm-populationBasis.html) extension SHALL be used to identify the result type of population criteria used in the measure
 2. Expressions used in population criteria SHALL return a value of the type specified by the populationBasis for the measure.
+3. If a DataRequirement is used to specify an allowed type for a population, the `type` SHALL be present and `profile`, and `codeFilter` elements MAY be present. Other elements SHALL NOT be present.
+   1. Add an invariant to the cqm-populationBases extension to enforce this conformance requirement.
```

Comment:
This shouldn't be language in the conformance requirement, it's a task to create this invariant in the computable measure profile on the use of the populationBasis extension there.